ITMO_FS/filters/univariate/UnivariateFilter.py

```diff
@@ -38,95 +37,27 @@ class UnivariateFilter(BaseEstimator, TransformerMixin, DataChecker):  # TODO AD
 n_repeated = 10, shuffle = False)
         >>> ufilter = UnivariateFilter(f_ratio_measure, select_k_best(10))
         >>> ufilter.fit(x, y)
+        >>> print(ufilter.selected_features_)
     """
 
-
-    def __init__(self, measure, cutting_rule=("Best by percentage",  0.2)):
-        # TODO Check measure and cutting_rule
-        super().__init__()
-        if type(measure) is str:
-            try:
-                self.measure = GLOB_MEASURE[measure]
```

Comment:
Why deleting this?

Reply:
this is moved to __apply_ms and is performed in fit as well